apps/explorer/src/components/network/Network.tsx

```diff
@@ -18,6 +19,12 @@ export default function WrappedNetworkSelect() {
         GROWTHBOOK_FEATURES.USE_TEST_NET_ENDPOINT
     ).on;
 
+    useEffect(() => {
```

Comment:
This should be fine for now but if we decide to add the network switch in another place as well, I think we will log the event twice. We could move it to the [useNetwork](https://github.com/MystenLabs/sui/blob/066bee63906008d84fed15963b33d1e14c094d9a/apps/explorer/src/context.ts#L18) hook that creates the context.

Reply:
> What's the issue with url params? What info is there that we don't have it the network variable from context? (assuming you refer to explorer)

By default, Plausible strips the URL params tracking only referrals. For us to track URL params, we have to use their manual tracking js file instead of `plausible-tracker `